ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift

```diff
@@ -97,6 +97,6 @@ class DemoListViewController: UITableViewController {
     }
 
     let cellReuseIdentifier: String = "TableViewCell"
-    private var isFirstLaunch: Bool = true
+    private static var isFirstLaunch: Bool = true
```

Comment:
what's the reason for this change?

Reply:
Without it, the isFirstLaunch bool keeps getting set back to true when going back to the Demo List View, but it should only be true on the first launch only. Changing it to static maintains the value once it gets set to false. This fixes the issue where the Color Demo View gets re-presented after tapping the "<" back button (video I sent you on Teams).